src/styles/main.css

```diff
@@ -207,6 +207,11 @@ html .lightModeOnly {
   visibility: hidden !important;
 }
 
+/* remove star sign on starred courses */
+.text-primary {
```

Comment:
Its probably not a good idea to hide this selector as it is a default color selector.
![image](https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/43612965/461cf433-a4da-46a6-b15f-8b9715741e15)
As you can see then it has hidden a button somewhere